README.md

```diff
@@ -442,6 +442,24 @@ export interface HonoServerOptions<E extends Env = BlankEnv> extends HonoServerO
    * For example, you can use this to bind `@hono/node-ws`'s `injectWebSocket`
    */
   onServe?: (server: ServerType) => void;
+  /**
+   * The Node.js Adapter rewrites the global Request/Response and uses a lightweight Request/Response to improve performance.
+   *
+   * If you this behavior, set it to `true`
```

Comment:
Just checking this because I think I'm running into this issue 😄 This sentence is missing a verb, I think 😊

Reply:
🙏